src/features/defi/providers/osmosis/components/OsmosisManager/Deposit/DepositReducer.ts

```diff
@@ -0,0 +1,48 @@
+import type { OsmosisDepositActions, OsmosisDepositState } from './DepositCommon'
+import { OsmosisDepositActionType } from './DepositCommon'
+
+export const initialState: OsmosisDepositState = {
+  txid: null,
+  opportunity: null,
+  poolData: null,
+  userAddress: null,
+  loading: false,
+  deposit: {
+    underlyingAsset0: {
+      amount: '',
+      denom: '',
+      fiatAmount: '',
+    },
+    underlyingAsset1: {
+      amount: '',
+      denom: '',
+      fiatAmount: '',
+    },
```

Comment:
Same thing, still reading through this PR but can we use selectors for these when we need them and get the denom/fiatAmount programmatically?

Reply:
👍